app/jobs/warm_up_quarterly_analytics_cache_job.rb

```diff
@@ -0,0 +1,15 @@
+class WarmUpQuarterlyAnalyticsCacheJob < ApplicationJob
+  queue_as :default
+
+  def perform
```

Comment:
Do you think it's better to write this as `[-2, -3, -4, -5].each`? I'm not sure if `-1 * (n + 1)` is obvious when reading.

Reply:
Agreed; just put the explicit values in the loop.